core/src/main/java/org/bitcoinj/protocols/channels/PaymentChannelServer.java

```diff
@@ -527,7 +527,7 @@ private void error(String message, Protos.Error.ErrorCode errorCode, CloseReason
         Protos.Error.Builder errorBuilder;
         errorBuilder = Protos.Error.newBuilder()
                 .setCode(errorCode)
-                .setExplanation(message);
```

Comment:
Hmm, explanation is optional in the protobuf. This means the better way would be to not set the explanation at all rather than set it to the empty string.